app/grandchallenge/reader_studies/utils.py

```diff
@@ -17,27 +33,34 @@ def migrate_reader_study_to_display_sets(rs, view_content):  # noqa: C901
             ds = DisplaySet.objects.create(reader_study=rs)
             images = []
             for key in item:
-                image = rs.images.get(name=item[key])
-                images.append(image.pk)
-
+                if "overlay" in key:
+                    continue
                 try:
-                    slug = view_content[key]
+                    slugs = view_content[key]
                 except KeyError:
                     raise ValueError(
                         f"No ComponentInterface provided for {key}."
                     )
-
-                try:
-                    ci = ComponentInterface.objects.get(slug=slug)
-                except ComponentInterfaceValue.DoesNotExist:
+                if len(slugs) == 0:
                     raise ValueError(
-                        f"ComponentInterface {slug} does not exist."
+                        f"No ComponentInterface provided for {key}."
                     )
-
-                civ, _ = ComponentInterfaceValue.objects.get_or_create(
-                    image=image, interface=ci
-                )
-                ds.values.add(civ)
+                else:
```

Comment:
This section needs a test.

Reply:
Everything but the ` if len(slugs) > 2:` is tested. Shall I add a test for that bit?